source/cppfront.cpp

```diff
@@ -4633,11 +4633,9 @@ class cppfront
                 // If explicit, upgrade deferred statements to pending
                 if (found_explicit_init)
                 {
-                    pending_statements.insert(
-                        pending_statements.end(),
-                        deferred_statements.begin(),
-                        deferred_statements.end()
-                    );
```

Comment:
You can make these moves and keep the `insert`.

                    pending_statements.insert(
                        pending_statements.end(),
                        std::make_move_iterator(deferred_statements.begin()),
                        std::make_move_iterator(deferred_statements.end());